osu.Game/Skinning/LegacySkin.cs

```diff
@@ -234,37 +235,50 @@ public void Dispose()
 
         private class LegacySpriteText : OsuSpriteText
         {
-            private readonly TextureStore textures;
-            private readonly string font;
+            private readonly LegacyGlyphStore glyphStore;
 
             public LegacySpriteText(TextureStore textures, string font)
             {
-                this.textures = textures;
-                this.font = font;
-
                 Shadow = false;
                 UseFullGlyphHeight = false;
-            }
 
-            protected override Texture GetTextureForCharacter(char c)
-            {
-                string textureName = $"{font}-{c}";
+                Font = new FontUsage(font, 16);
```

Comment:
where does 16 come from?

Reply:
Can be removed.

Reply:
Actually I take that back since the text builder isn't overridden with a custom size. It's just what it was previously (should be OsuFont.DEFAULT_FONT_SIZE).

Reply:
so `Font = OsuFont.Default.With(font);` should be safest?

Reply:
That would also work fine, yeah.